src/releaseBusV2/release-bus-v2.production-authority.ts

```
@@ -0,0 +1,1774 @@
+  ): Promise<ReleaseBusV2ProductionAuthorityCompletionResponse> {
+    return this.finish(input, 'FAILED', input.reason_code);
+  }
+
+  private async finish(
+    input:
+      | ReleaseBusV2ProductionAuthorityReauthorizeInput
+      | ReleaseBusV2ProductionAuthorityFailureInput,
+    status: 'COMPLETED' | 'FAILED',
+    failureCode:
+      | ReleaseBusV2ProductionAuthorityFailureCode
+      | undefined = undefined,
```

ESLint

[error] Replace `|·ReleaseBusV2ProductionAuthorityFailureCode⏎·····` with `ReleaseBusV2ProductionAuthorityFailureCode`

(prettier/prettier)